src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -2146,6 +2160,8 @@ ReadMachineTypeFromPEHeader readMachineTypeFromPEHeader
                         assemblyMetadataCache
                         );
 
+                    dependencyTable.FindDependenciesOfExternallyResolvedReferences = FindDependenciesOfExternallyResolvedReferences;
```

Comment:
Quite the large constructor parameter list right? :)

Reply:
Yeah. Are you OK with this deviation?

Reply:
Yes, just for the sake of less churn. The large number of arguments seems bad enough and shadows this deviation :)